docs/website/layouts/partials/docs/dashboard.html

```diff
@@ -1,8 +1,7 @@
-{{ $localMode  := eq (getenv "ENV") "local" }}
 {{ $home       := site.BaseURL }}
 {{ $logo       := site.Params.logos.sidebar | relURL }}
 {{ $releases   := site.Data.releases }}
-{{ $latest     := printf "v%s" (index $releases 0) }}
+{{ $latest     := printf "%s" (index $releases 0) }}
```

Comment:
This can just be `index $releases 0`, as there's no longer any interpolation being performed

Reply:
Good catch